Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -294,25 +294,11 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
       Double?,
       PledgeSummaryViewData
     ) in
-    guard let projectCurrencyCountry = project.stats.currentCountry else {
-      return (
-        PostCampaignRewardsSummaryViewData(
-          rewards: rewards,
-          selectedQuantities: selectedQuantities,
-          projectCountry: project.country,
-          omitCurrencyCode: project.stats.omitUSCurrencyCode,
-          shipping: shippingSummaryData
-        ),
-        bonusOrPledgeUpdatedAmount,
-        pledgeTotalSummaryData
-      )
-    }
-
-    return (
+    (
       PostCampaignRewardsSummaryViewData(
         rewards: rewards,
         selectedQuantities: selectedQuantities,
-        projectCountry: projectCurrencyCountry,
+        projectCountry: project.country,
```

Comment:
Huh, interesting. What's the difference between `project.country` and `project.stats.currentCountry`?

Reply:
`currentCountry` is based on the user's location! It's what we use to determine what to convert currency to, for example. It's the opposite of what we want for the actual project's country